.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+---
+# When a tag is created
+# - build and publish container to quay
+# - publish role to ansible-galaxy
+name: Release a new version, publish artefacts
+
+on:
+  push:
+    tags:
+      - "v*"
+    branches:
+      - "main"
+
+# https://github.com/marketplace/actions/push-to-registry
+jobs:
+  publish:
+    name: Build UMD4 for CentOS 7
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        umd_version:
+          - "umd4"
+        os:
+          - "centos7"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build umd4-centos7 image with molecule
+        id: build-image
+        uses: gofrolist/molecule-action@v2
+        with:
+          molecule_command: create
+          molecule_args: --scenario-name ${{ matrix.umd_version }}
+        env:
+          ANSIBLE_FORCE_COLOR: "1"
+
+      # TODO look into pushing some description
+      # https://github.com/marketplace/actions/update-container-description-action
+      - name: Publish image to Quay
+        run: |
+          docker commit umd-${{ matrix.os }} quay.io/egi/${{ matrix.umd_version }}:${{ matrix.os }}
```

Comment:
Should there be also a quay.io/egi/${{ matrix.umd_version }}:latest ?

Reply:
In fact the repo got created to have tags corresponding to the OS release, like `umd4:centos6` or `umd4:centos7`. `umd4:centos6` is deprecated, but you get the idea, and at some point we may have `umd5:centos7` and `umd5:centos9` so I'm not sure to what should point `latest`.
We could also totally change how it's managed/organised, it's the perfect time to do this.

Reply:
The problem I see is:
```
$ docker pull quay.io/egi/umd4
Using default tag: latest
Error response from daemon: manifest for quay.io/egi/umd4:latest not found: manifest unknown: manifest unknown
```

we may be ok with that anyway. I would point `latest` to what we believe is the recommended image to run, for `umd4` that should pointing to `centos7`

Reply:
Indeed ware may be OK with this, as the usage instructions are/should be `docker pull quay.io/egi/umd4:centos7`.
That said I'm fine with having `latest` point to an arbitrary release that we recommend.
I'm not sure how to implement this, hardcoding the push to latest in the GitHub action for the specific version? What do you think/suggest?

Reply:
With the current structure of the job, I'm not sure how to make this simple to tag just one image as `latest` so let's skip it for the time being. Let people read documentation :)

Reply:
Fine, we can look into this later, if it happens that someone wants to use those images ;)